### PR TITLE
feat(gen): kerberos_renewal_principals defines resource by destination file

### PR DESCRIPTION
**Pull request by sarkapalkovicova**
Split resource data to separate files by using attribute kerberosPrincipalsFileSuffix.

### CODE REVIEW

File: slave/process-kerberos-renewal-principals/bin/process-kerberos_renewal_principals.sh [hunk 1]
```diff
@@ -1,25 +1,45 @@
...
+	E_FINISHED_WITH_ERRORS=(50 'Slave script finished with errors!')
+
+	ERROR=0
 
 	create_lock
 
-	cp "${FROM_PERUN}" "${DST_DIR}/${DST_FILE}"
+	# Delete all files from destination directory
+	if [[ $(find $DST_DIR -mindepth 1 -maxdepth 1) ]]; then
+		if ! rm "$DST_DIR"*; then
```
Comment (zlamalp) [inline slave/process-kerberos-renewal-principals/bin/process-kerberos_renewal_principals.sh:19]: I'm afraid, that destination folder contains also other files of service configuration, so we should only delete files with `kerberos_renewal_principals` prefix in the destination folder.

**Review by zlamalp — CHANGES_REQUESTED** [review]

**Comment by Johaney-s** [comment]
one more thing i just realized - at some point we decided to give commit scopes for perun-services repo by the name of the service, not gen/send/slave. please update it in the commit message, i'll update the documentation

**Review by Johaney-s — APPROVED** [review]

File: slave/process-kerberos-renewal-principals/changelog [hunk 2]
```diff
@@ -1,3 +1,10 @@
+perun-slave-process-kerberos-renewal-principals (3.0.1) stable; urgency=low
+
+  * Handled receive of many files from perun.
+  * Script copies them to the destination directory and deletes unnecessary ones.
```
Comment (zlamalp) [inline slave/process-kerberos-renewal-principals/changelog:4]: This line is too long, changelog "content" lines are limited to 80 chars.

**Review by zlamalp — CHANGES_REQUESTED** [review]
Also please squash the commits into single feat commit.

**Review by zlamalp — APPROVED** [review]

**Pull request merged**